src/server/systems/NetworkingServer.hpp

```diff
@@ -94,6 +98,17 @@ class NetworkingServer {
     std::mutex m_playerUnreliableUpdates_lock;
     std::vector<InputDef::PlayerUnreliableInput> m_playerUnreliableUpdates_t;
 
+    std::mutex m_broadcastGameStateMetric_lock;;
+    TemporalMetric m_broadcastGameStateMetric_t{30, sf::seconds(0.5f)};
+
+    std::mutex m_playerIdsToUpdatesMetric_lock;
+    std::unordered_map<sf::Uint32, TemporalMetric*> m_playerIdsToUpdatesMetric_t;
+    std::unordered_map<sf::Uint32, float> m_playerIdsToUpdatesRates_t;
+
+    std::mutex m_playerIdsToDriftMetrics_lock;
+    std::unordered_map<sf::Uint32, TemporalMetric*> m_playerIdsToDriftMetrics_t;
+    std::unordered_map<sf::Uint32, float> m_playerIdsToDrifts_t;
```

Comment:
I compute into this float unordered_map which is then returned to ServerGameController (and different thread) to avoid thread safety issues on the TemporalMetric objects